### PR TITLE
Report 50th, 90th and 99th percentailes of network programming latency.

### PR DESCRIPTION
**Pull request by oxddr**
refs https://github.com/kubernetes/perf-tests/issues/575

/label do-not-merge/work-in-progress

### CODE REVIEW

**Comment by mm4tt** [comment]
Let me know when this is ready for review

**Comment by oxddr** [comment]
I need to test it, but I want https://github.com/kubernetes/test-infra/pull/13272 to be merged first.

File: clusterloader2/pkg/measurement/common/slos/network_programming.go [hunk 1]
```diff
@@ -35,7 +36,7 @@ const (
 	// %v should be replaced with query window size (duration of the test).
 	// This measurement assumes, that there is no data points for the rest of the cluster-day.
 	// Definition: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/network_programming_latency.md
-	query = "quantile_over_time(0.99, kubeproxy:kubeproxy_network_programming_duration:histogram_quantile{quantile='0.99'}[%v])"
+	query = "quantile_over_time(0.99, kubeproxy:kubeproxy_network_programming_duration:histogram_quantile{}[%v])"
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/common/slos/network_programming.go]: Just to confirm, what we'll have in perf-dash will be 
`99th percentile of all measurements of Xth percentiles last 5min, where X=50,90,99`

Right?

Comment (oxddr) [reply clusterloader2/pkg/measurement/common/slos/network_programming.go]: Yes, that's right. 

Shall I update the comment to be more clear?

Comment (mm4tt) [reply clusterloader2/pkg/measurement/common/slos/network_programming.go]: Would be great, but can be done in a follow up PR. This looks good, don't want to block it anymore

**Pull request merged**